src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+/**
+ * Parses different strings and acts as a helper for other classes.
+ */
+public class Parser {
+
+    /**
+     * Returns the type of command given a command from a user.
+     *
+     * @param str The original command from the user.
+     * @return Type of command.
+     */
+    public String parseCommand(String str) {
+        if (str.equals("bye")) {
+            return "bye";
+        } else if (str.equals("list")) {
+            return "list";
+        } else if (str.startsWith("done")) {
+            return "done";
+        } else if (str.startsWith("delete")) {
+            return "delete";
+        } else if (str.startsWith("todo") || str.startsWith("event") || str.startsWith("deadline")) {
+            return "addTask";
+        } else if (str.startsWith("find")) {
+            return "find";
+        } else if (str.equals("\n")){
```

Comment:
Should this { be surrounded with whitespace? I noticed the same in various other places too.